politeiawww/backend.go

```diff
@@ -183,6 +183,11 @@ func checkSignature(id []byte, signature string, elements ...string) error {
 	return nil
 }
 
+func formatUsername(username string) string {
+	u := strings.TrimSpace(username)
```

Comment:
Why are you creating a variable here?

Just do: return strings.ToLower(strings.TrimSpace(username))

Reply:
Fixed.